README.md

```diff
@@ -120,13 +122,40 @@ InvisibleCaptcha.setup do |config|
   # config.timestamp_threshold = 4
   # config.timestamp_enabled   = true
   # config.injectable_styles   = false
+  # config.ip_enabled          = true
+  # config.secret              = REQUIRED_SECRET_KEY
 
   # Leave these unset if you want to use I18n (see below)
   # config.sentence_for_humans     = 'If you are a human, ignore this field'
   # config.timestamp_error_message = 'Sorry, that was too quick! Please resubmit.'
 end
 ```
 
+#### IP Enabled Setup
+
+View code:
+
+```erb
+
+<%= form_for(@topic) do |f| %>
+  <%= f.invisible_captcha :subtitle, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
+  <!-- or -->
+  <%= invisible_captcha :subtitle, :topic, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
+<% end %>
+```
+
+Controller code:
+
+```ruby
+class TopicsController < ApplicationController
+  # By default, invisible_captcha sets instance variabls for all actions,
+  # but does not run detect_spam for actions specified.
+  _
+  invisible_captcha only: [:new, :create, :update], honeypot: :subtitle
```

Comment:
sorry, this is really weird API and can potentially break current people code. I don't want to break people app with sensible stuff like forms, sign-ups, ...

I still think we should find a way to make all this new "spinner" thing under the hood without the users updating and changing all current `invisible_captcha` occurrences in their apps.